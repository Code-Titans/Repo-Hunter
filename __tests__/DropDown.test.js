import React from 'react';
import { mount } from 'enzyme';

import DropDown from '../components/Dropdown';

const sorting = [
  { value: 'Most', label: 'Most', name: 'sort' },
  { value: 'Least', label: 'Least', name: 'sort' },
];

describe('DropDown component', () => {
  let wrapper;
  const props = {
    value: '',
    defaultInputValue: '',
    onChange: jest.fn(),
    instanceId: 'kimame',
    options: sorting,
  };
  beforeEach(() => {
    // eslint-disable-next-line react/jsx-props-no-spreading
    wrapper = mount(<DropDown {...props} />);
  });

  it('should render DropDown component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
