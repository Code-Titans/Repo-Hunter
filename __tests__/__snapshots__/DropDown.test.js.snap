// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropDown component should render DropDown component 1`] = `ReactWrapper {}`;
